### DISCUSSION TITLE
Unable to find the required Inet version

### DISCUSSION THREAD

**Discussion by CC-vga** [CoRE-RG/CoRE4INET#32] [Q&A] [answered]
Hello, 

I would need to instal CoRE4INET and I'm not able to find the Inet version suggested on your README. For Omnet++ 5.5.1 the recommended Inet version is 4.1.2. But with this version of Inet, I'm not able to build CoRE4INET.
On the other hand, on the Inet download page, I cannot find the version 3.6.6.
Can you help me please to find the required version of Inet? I'm building CoRE4INET from CLI, so a link to the archive would be helpful.

All best
Voica

**Comment by Musteblume** [comment, marked as answer]
Hi,

you could clone or download from the inet github repo: https://github.com/inet-framework/inet/releases/tag/v3.6.6

Best regards

**Comment by CC-vga** [comment]
Got it thanks!
